client/account-recovery/reset-password/reset-password-form/index.jsx

```diff
@@ -15,31 +15,46 @@ import FormFieldset from 'components/forms/form-fieldset';
 import FormLegend from 'components/forms/form-legend';
 import ResetOptionSet from './reset-option-set';
 
-import { getAccountRecoveryResetOptions } from 'state/selectors';
+import {
+	pickResetMethod,
+	requestReset,
+} from 'state/account-recovery/reset/actions';
+
+import {
+	getAccountRecoveryResetUserData,
+	getAccountRecoveryResetOptions,
+	getAccountRecoveryResetPickedMethod,
+	getAccountRecoveryResetRequestError,
+	isRequestingAccountRecoveryReset,
+} from 'state/selectors';
 
 export class ResetPasswordFormComponent extends Component {
 	static defaultProps = {
 		translate: identity,
+		pickedMethod: null,
+		userData: {},
+		requestError: null,
+		isRequesting: false,
```

Comment:
I think you are right. Those defaults defeat the purpose of `isRequired`, hence been removed by 337137104eb9a3bc47d69ccba012882a45764080.